src/Pages/Music/Videos/Videoid_Int.elm

```diff
@@ -34,19 +47,30 @@ type alias Model =
     , video : Maybe VideoObj
     , album_list : List AlbumObj
     , video_list : List VideoObj
+    , prepareDownloadPath : Maybe String
+    , modalstate : VideoModal
     }
 
 
 init : Shared.Model -> Url Params -> ( Model, Cmd Msg )
 init shared { params } =
-    ( { videoid = params.videoid
-      , video = getVideos params.videoid shared.video_list
-      , album_list = shared.album_list
-      , artist_list = shared.artist_list
-      , video_list = shared.video_list
-      }
-    , Cmd.none
-    )
+    let
+        sharedobjects =
```

Comment:
Not a fan of this naming, maybe `initialState`?

Reply:
Or better `initialModel`

Reply:
Done :+1: